libvast/vast/schema.hpp

```diff
@@ -101,18 +102,21 @@ get_schema_dirs(const caf::actor_system_config& cfg,
                 std::vector<const void*> objpath_addresses = {nullptr});
 
 /// Loads a single schema file.
-/// @param sf The file path.
+/// @param schema_file The file path.
 /// @returns The parsed schema.
-caf::expected<schema> load_schema(const path& sf);
+caf::expected<schema> load_schema(const path& schema_file);
 
 /// Loads *.schema files from the given directories.
 /// @param schema_dirs The directories to load schemas from.
+/// @param max_recursion The maximum number of nested directories before
+///        aborting.
```

Comment:
```suggestion
/// aborting.
```

My clang-format does not add this indentation. It's just kept because you added it manually, but that only causes trouble when we edit the text later on. The manual indentation also looks really weird with the hover definition info from clangd.

Reply:
This is still the way we have it our contribution guidelines: https://github.com/tenzir/.github/blob/master/contributing.md#comments. I'd say we do one consolidated fixup if we want to change that style.

Reply:
I'd say our clang-format config file has a higher authority than our contribution guidelines, simply because it cannot be ignored.

Reply:
Fair enough; changed.